src/styles/components/_doc-sidebar.scss

```diff
@@ -28,7 +28,7 @@ html[data-theme='dark'] {
 
 //overrides
 #__docusaurus {
-  [class^='docSidebarContainer'] {
+  aside > div {
```

Comment:
The sidebar border is incorrect

| main | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/266df152-9dbd-45a6-81ed-2892ee5c4fb5) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/e046beed-e55d-4182-b32e-e9c464388ecd) |

Should be using `var(--sidebar-border-c);` instead of `var(--ifm-toc-border-color);`. This should also be using `border-inline-end` instead of `border-right` for RTL.

This selector change may be to blame here. The correct border is applied to the div inside of `aside`, but the `aside` itself also has a border.